### DISCUSSION TITLE
Pushing Changes to Repository using External Application (Ignition) and a Powershell Script

### DISCUSSION THREAD

**Discussion by jguagnelli2023** [community/community#69225] [New to GitHub]
Good afternoon, 

I am trying to commit and push changes to my repository calling a powershell script. This script is working when running it directly from Powershell and psuhing the changes. The problem is when I call the script from Ignition, the script  runs but it does not commit or pushes to my repo. This script is authenticating using PAT. Am I missing something?

**Comment by queenofcorgis** [comment]
Hi there - I'm not too familiar with Ignition, have you posed this question on their forum or within the project's repository?

**Reply by Alcatraz-Dev** [reply]
try at Delete repository and make new one
